app/views/layouts/application.html.erb

```diff
@@ -56,13 +56,17 @@
 
     <div class="mainContentWrapper">
       <div class="mainContentContainer">
+
```

Comment:
This is great. Bootstrap gave us cool stuf, you used that cool stuff to design a nice messaging interface.